opensplice_cmake_module/config/ros_ospl.xml

```diff
@@ -21,5 +21,13 @@
             <!-- the following one is necessary only for TwinOaks CoreDX DDS compatibility -->
             <!-- <ExplicitlyPublishQosSetToDefault>true</ExplicitlyPublishQosSetToDefault> -->
         </Compatibility>
+        <Discovery>
+             <!-- By default, OpenSplice only allows up to 9 participants (see
+                  http://download.prismtech.com/docs/Vortex/html/ospl/ConfGuide/guide.html#maxautoparticipantindex)
+                  This means that only 9 nodes can be run at a time, which even
+                  moderately complex robots easily achieve.
```

Comment:
I think the limit is 9 participants per machine - not overall?

Reply:
Good point; as far as I understand, it is a limitation of 9 per machine, not overall.  I've update the comment to reflect that in 94ccaa2e4811890f9f5cabbaf072d813fb571a22